phpstan-baseline.neon

```diff
@@ -1705,11 +1705,6 @@ parameters:
 			count: 2
 			path: src/PrestaShopBundle/Model/Product/AdminModelAdapter.php
 
-		-
-			message: "#^Namespace Configuration is forbidden, No legacy calls inside the prestashop bundle\\. Please create an interface and an adapter if you need to\\.$#"
-			count: 2
-			path: src/PrestaShopBundle/Model/Product/AdminModelAdapter.php
-
```

Comment:
this one was introduced here for a reason, you can't just delete it because it fails :smile:.
Instead of deleting this rule, use ShopConfigurationInterface inside AdminModelAdapter, that should solve the problem.